v63005/u_inref/gtm8877.csh

```diff
@@ -112,7 +112,9 @@ chmod -w $ydb_dist/restrict.txt
 echo "# No label specified in filters"
 $ydb_dist/mumps -run zsystemfn^gtm8877
 $ydb_dist/mumps -run pipeopenfn^gtm8877
-echo "# Checking th Sys Log"
+echo "# Checking the Sys Log"
+# getoper.csh calls an m script, which creates excess RESTRICTSYNTAX errors in the syslog
```

Comment:
Reword as "Avoid extra RESTRICTSYNTAX errors from getoper.csh (which does mumps -run) by removing restrict.txt before the call"